src/zenml/integrations/aws/flavors/sagemaker_orchestrator_flavor.py

```diff
@@ -74,11 +84,13 @@ class SagemakerOrchestratorSettings(BaseSettings):
                     Data must be available locally in /opt/ml/processing/output/data/<ChannelName>.
     """
 
-    instance_type: str = "ml.t3.medium"
+    instance_type: Optional[str] = None
     processor_role: Optional[str] = None
     volume_size_in_gb: int = 30
     max_runtime_in_seconds: int = 86400
     processor_tags: Dict[str, str] = {}
```

Comment:
This and `processor_args` should maybe be renamed/deprecated now that its more generic?

Reply:
Reworked as suggested